ext/iperf3/Kconfig

```diff
@@ -0,0 +1,48 @@
+#
+# Copyright (c) 2020 Nordic Semiconductor
+#
+# SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+#
+
+# iperf3 NRF integration
+
+config  NRF_IPERF3
+	bool "Iperf3 NRF integration"
+	default n
```

Comment:
No need for `default n`, it's implicit behavior.

Reply:
Removed